app/views/main.scala.html

```diff
@@ -47,6 +48,6 @@
         @fragments.global.footer(product, edition, contactUsCountry)
         @fragments.footerJavaScript()
         <!--AWIN affiliate tracking tag - see here: https://drive.google.com/drive/folders/19AOGPkPbbFjqPJTdCcCba3EwpLiSHBwW -->
-         <script src="https://www.dwin1.com/11152.js" type="text/javascript" defer="defer"></script>
+         <script src="@hashedPathFor("javascripts/vendor/11152.js")" type="text/javascript" defer="defer"></script>
```

Comment:
Could we use a more obvious filename for this self hosted asset to make it obvious what vendor its for?